scvi/models/vae.py

```diff
@@ -216,7 +216,7 @@ def get_reconstruction_loss(
             )
         elif self.reconstruction_loss == "nb":
             reconst_loss = (
-                -NegativeBinomial(mu=px_rate, theta=px_r).log_prob(x).sum(dim=-1)
+                -NegativeBinomial(mu=px_rate, theta=px_r).log_prob(x).mean(dim=-1)
```

Comment:
This will change a lot of things, the loss is averaged in the `loss` function in the trainer.

Reply:
ahh, I see. ok, then this change probably can be reverted then